rpi5.yaml

```diff
@@ -342,3 +345,24 @@ parameters:
                 type: raw_image
                 size: 8 GiB
                 image_path: "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/%{XT_DOMD_IMAGE}-%{MACHINE}.rootfs.ext4"
+
+    nvme:
+      overrides:
+        components:
+          domd:
+            builder:
+              conf:
+                - [DOMD_OVERLAYS, "%{SOC_FAMILY}-%{MACHINE}-pcie1.dtbo"]
+                - [MACHINE_FEATURES:append, " domd_nvme"]
+                - [DOMD_BOOTARGS, "console=ttyAMA0 earlycon=xen earlyprintk=xen clk_ignore_unused root=\\\\/dev\\\\/nvme0n1p1 rootfstype=ext4 rootwait"]
+        images:
+          rootfs:
```

Comment:
So. How data should appear on nvme? This info should be in readme

Reply:
Updated readme